src/views/components-demo/dropzone.vue

```diff
@@ -0,0 +1,41 @@
+<template>
+  <div class="components-container">
+    <aside>
+      Based on
+      <a
+        class="link-type"
+        href="https://github.com/rowanwins/vue-dropzone"
+      >dropzone</a>
+      .
+      {{ $t('components.dropzoneTips') }}
+    </aside>
+    <div class="editor-container">
+      <Dropzonec
```

Comment:
```suggestion
      <dropzonec
```
prefer xxx-xxx format